source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -268,21 +283,98 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         }
     }
 
+    winrt::fire_and_forget ConfigureSvgImageSourceAsync(winrt::AdaptiveRenderContext renderContext,
+                                                        winrt::hstring url,
+                                                        winrt::Uri uri,
+                                                        winrt::SvgImageSource svgImageSource,
+                                                        winrt::TileControl tileControl)
+    {
+        auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
+        auto weakImageSource{winrt::make_weak(svgImageSource)};
+        auto weakTileControl{winrt::make_weak(tileControl)};
+
+        if (imageUrl.SchemeName() == L"data")
+        {
```

Comment:
Can you add resume_background call here, and you can make StoreAsync() call to sync by calling .get() on it.

Reply:
As discussed offline, leaving it as is for now.